jbpm-db-scripts/src/test/java/org/jbpm/persistence/scripts/GenerateDDLScriptsTests.java

```diff
@@ -36,33 +35,38 @@
  * Utility class for generating DDL scripts (create and drop) please ignore it.
  */
 @RunWith(Parameterized.class)
-@Ignore
 public class GenerateDDLScriptsTests {
 
-    private static class ScriptFile {
+    static {
+        System.setProperty("org.kie.persistence.postgresql.useBytea", "false");
+    }
+
+    public static class ScriptFile {
 
         private String dialect;
         private String alias;
         private String prefix;
+        private String subtype;
         private boolean newGenerator;
 
         public ScriptFile(String dialect, String alias, boolean newGenerator) {
-            this(dialect, alias, alias, newGenerator);
+            this(dialect, alias, alias, (newGenerator ? "new-" : ""), newGenerator);
```

Comment:
Shouldn't it be "springboot-" subtype instead of "new-"?

Reply:
done